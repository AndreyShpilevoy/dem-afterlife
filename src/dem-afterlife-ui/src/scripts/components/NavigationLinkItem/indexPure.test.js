/* eslint no-undef: 0, fp/no-unused-expression: 0, fp/no-nil: 0*/

import React from 'react';
import {shallow} from 'enzyme';
import {NavigationLinkItemPure} from './index';

describe('NavigationLinkItem Pure', () => {
    const styles = {
        '.link-0-0:hover': 'link-0-0',
        link: 'link-0-0'
    };

    it('component match expected snapshot', () => {
        const navigationLinkItem = {id: 1, title: 'Conference', href: '/', order: 1};
        expect(shallow(<NavigationLinkItemPure styles={styles} navigationLinkItem={navigationLinkItem}/>)).toMatchSnapshot();
    });
});
