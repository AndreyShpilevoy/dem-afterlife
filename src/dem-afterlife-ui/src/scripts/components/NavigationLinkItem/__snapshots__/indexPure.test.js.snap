// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationLinkItem Pure component match expected snapshot 1`] = `
<li>
  <withStyles(LinkPure)
    className="link-0-0"
    to="/"
  >
    Conference
  </withStyles(LinkPure)>
</li>
`;
